spark/v3.1/spark/src/main/java/com/netease/arctic/spark/writer/KeyedSparkBatchWrite.java

```diff
@@ -63,12 +64,17 @@ public class KeyedSparkBatchWrite implements ArcticSparkWriteBuilder.ArcticWrite
   private final long txId;
   private final String hiveSubdirectory;
 
-  KeyedSparkBatchWrite(KeyedTable table, StructType dsSchema) {
+  private final boolean orderedWriter;
+
+  KeyedSparkBatchWrite(KeyedTable table, LogicalWriteInfo info) {
     this.table = table;
-    this.dsSchema = dsSchema;
+    this.dsSchema = info.schema();
     this.legacyTxId = table.beginTransaction(null);
     this.txId = TablePropertyUtil.allocateTransactionId(table.asKeyedTable());
     this.hiveSubdirectory = HiveTableUtil.newHiveSubdirectory(this.legacyTxId);
+    this.orderedWriter = Boolean.parseBoolean(info.options().getOrDefault(
```

Comment:
How should the user change this property?
Is the default `true` value is  appropriate？

Reply:
The user can't change this property.  do a local sort before write is forced.